### PR TITLE
PR : Dev/feriel to main 06/04/2021 15:39

### PR DESCRIPTION
**Pull request by feriel27**
## List of changes 
- update method findCompteById

### CODE REVIEW

File: src/main/java/tn/windata/demo/service/ICompteService.java [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
+import java.util.Optional;
+
 import tn.windata.demo.entities.Compte;
 
 public interface ICompteService {
 
 	Compte addCompte(Compte compte);
-	Compte getCompteById(Long id);
+	Optional<Compte>  getCompteById(Long id);
```
Comment (ahmed-makni) [inline src/main/java/tn/windata/demo/service/ICompteService.java]: make return Type Compte not Optional

File: src/main/java/tn/windata/demo/service/CompteServiceImpl.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public Compte addCompte(Compte compte) {
 
 	}
 	@Override
-	public Compte getCompteById (Long id) {
-		return iCompteRepo.getCompteById( id);
+	public Optional<Compte> getCompteById (Long id) {
```
Comment (ahmed-makni) [inline src/main/java/tn/windata/demo/service/CompteServiceImpl.java]: make return Type Compte

File: src/main/java/tn/windata/demo/service/CompteServiceImpl.java [hunk 3]
```diff
@@ -23,7 +23,7 @@ public Compte addCompte(Compte compte) {
...
+		return iCompteRepo.findById( id);
```
Comment (ahmed-makni) [inline src/main/java/tn/windata/demo/service/CompteServiceImpl.java]: add .get() methode after findById( id)

**Review by ahmed-makni — COMMENTED** [review]
check comments

File: src/main/java/tn/windata/demo/service/CompteServiceImpl.java (hunk 3 above)
Comment (feriel27) [reply src/main/java/tn/windata/demo/service/CompteServiceImpl.java]: resolved

**Pull request merged**